addon/components/file-dropzone/component.js

```diff
@@ -80,6 +80,51 @@ export default BaseComponent.extend({
   active: false,
   valid: true,
 
+  /**
+    Whether multiple files can be selected when uploading.
+    @argument multiple
+    @type {boolean}
+   */
+  multiple: null,
```

Comment:
If we have to duplicate the documentation to make YUIDocs happy, do we also have to duplicate the default values? If so we should remove them (`multiple`, `name`, `disabled`, `accept`, `onfileadd`) from `BaseComponent` as they are overriden by both child classes.

Reply:
Staged in #287